README.md

```diff
@@ -60,7 +60,11 @@ function App() {
   return (
     <SearchHeadlessProvider searcher={searcher}>
       <SearchBar />
-      <UniversalResults />
+      <UniversalResults verticalConfigMap={{
```

Comment:
Can we have verticalConfigMap outside of the function? Since we've been following the react-perf guidelines even though they're debatable